kafka-ui-react-app/src/tests/Topics/Details/Messages/Messages.spec.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import JSONTree from 'react-json-tree';
+import * as useDebounce from 'use-debounce';
+import DatePicker from 'react-datepicker';
+import Messages, {
+  Props,
+} from '../../../../components/Topics/Details/Messages/Messages';
+import PageLoader from '../../../../components/common/PageLoader/PageLoader';
+
+describe('Messages', () => {
+  beforeEach(() => {
+    jest.restoreAllMocks();
+  });
+
+  const createMessageComponent = (props: Partial<Props> = {}) =>
+    shallow(
+      <Messages
+        clusterName="Test cluster"
+        topicName="Cluster topic"
+        isFetched
+        fetchTopicMessages={jest.fn()}
+        messages={[]}
+        partitions={[]}
+        {...props}
+      />
+    );
+
+  it('Messages component should render PageLoader', () => {
```

Comment:
This should read like a sentence. `It renders PageLoader`
```suggestion
  it('renders PageLoader', () => {
```